docs/website/docs/dlt-ecosystem/verified-sources/openapi-generator.md

```diff
@@ -0,0 +1,198 @@
+# 4. If you have any kind of authentication on your pipeline (this example does not), 
+#    open the `.dlt/secrets.toml` and provide the credentials. You can find further 
+#    settings in the `.dlt/config.toml`.
+
+# 5. Go to the created pipeline folder and run your pipeline.
+$ cd pokemon-pipeline
+$ PROGRESS=enlighten python pipeline.py # we use enlighten for a nice progress bar :)
+
+# 6. Print the pipeline info to the console to see what got loaded.
+$ dlt pipeline pokemon_pipeline info
+
+# 7. You can now also install Streamlit to see a preview of the data; you should 
+#    have loaded 40 Pokemons and their details.
+$ pip install pandas streamlit
+$ dlt pipeline pokemon_pipeline show
+
+# 8. You can go to our docs at https://dlthub.com/docs to learn how to modify 
+#    the generated pipeline to load to many destinations, place schema contracts 
+#    on your pipeline, and many other things.
+
+# NOTE: We used the `--global-limit 2` CLI flag to limit the requests to the PokeAPI 
+#       for this example. This way, the Pokemon collection endpoint only gets queried 
+#       twice, resulting in 2 x 20 Pokemon details being rendered.
+```
+
+## What will be created?
+
+When you run the `init` command above, the following files will be generated:
+
+```text
+pokemon_pipeline/
+├── .dlt/
+│   ├── config.toml     # dlt config, learn more at dlthub.com/docs
+│   └── secrets.toml    # your secrets, only needed for APIs with auth
+├── pokemon/
+│   └── __init__.py     # your rest_api dictionary, learn more below
+├── rest_api/
+│   └── ...             # rest_api copied from our verified sources repo
+├── .gitignore
+├── pokemon_pipeline.py # your pipeline file that you can execute
+├── README.md           # a list of your endpoints with some additional info
+└── requirements.txt    # the pip requirements for your pipeline
+```
+
+> If you re-generate your pipeline, you will be prompted to continue if this folder exists. If you select yes, all generated files will be overwritten. All other files you may have created will remain in this folder.
+
+## A closer look at your rest_api dictionary in pokemon/__init__.py
+
+This file contains the configuration dictionary for the [dlt rest_api](https://dlthub.com/docs/dlt-ecosystem/verified-sources/rest_api) source which is the main result of running this generator. For our Pokemon example, we have used an OpenAPI 3 spec that works out of the box. The result of this dictionary depends on the quality of the spec you are using, whether the API you are querying actually adheres to this spec, and whether our heuristics manage to find the right values. You can edit this file to adapt the behavior of the dlt rest_api accordingly. Please read our [dlt rest_api](https://dlthub.com/docs/dlt-ecosystem/verified-sources/rest_api) docs to learn how to do this and play with our detailed [Google Colab example](https://colab.research.google.com/drive/1MRZvguOTZj1MlkEGzjiso8lQ_wr1MJRI?usp=sharing#scrollTo=LHGxzf1Ev_yr).
```

Comment:
```suggestion
This file contains the [configuration dictionary](./rest_api#source-configuration) for the rest_api source which is the main result of running this generator. For our Pokemon example, we have used an OpenAPI 3 spec that works out of the box. The result of this dictionary depends on the quality of the spec you are using, whether the API you are querying actually adheres to this spec, and whether our heuristics manage to find the right values. You can edit this file to adapt the behavior of the dlt rest_api accordingly. Read our [dlt rest_api](./rest_api) docs to learn how to configure rest_api source and play with our detailed [Google Colab example](https://colab.research.google.com/drive/1MRZvguOTZj1MlkEGzjiso8lQ_wr1MJRI?usp=sharing#scrollTo=LHGxzf1Ev_yr).
```

Linked config docs directly using a relative link + made call to action more specific.